### PR TITLE
DOC-7263 - Implementing CrazyEgg

### PR DESCRIPTION
**Pull request by x8a**
Added crazyegg script to SEO component for the following pages:
- https://docs.newrelic.com
- https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/create-your-new-relic-account/

### CODE REVIEW

File: src/components/SEO.js [hunk 1]
```diff
@@ -10,6 +10,22 @@ const METADATA = [
   },
 ];
 
+const crazyEgg = (location) => {
+  const homepage = 'https://docs.newrelic.com';
+  const signup =
+    'https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/create-your-new-relic-account';
+
+  if (location === homepage || signup) {
```
Comment (zstix) [inline src/components/SEO.js:18]: `location` is a little different from [`window.location`](https://developer.mozilla.org/en-US/docs/Web/API/Window/location) - it's not a plain string, but rather an object (you can see that in the prop-types below). Our [documentation](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#seo) on the `SEO` component doesn't specifiy the _exact_ shape of that object, but it does say that it should have a `pathname` field that we can use for this purpose.

Additionally, javascriprt will treat the `|| signup` as a separate condition on it's own and not attempt to verify if `signup` is equal to `location`. You need to supply `location ===` for both statements.

Here's the adjustments to the code that should handle both those things:

```suggestion
  const { pathname } = location;
  
  const homepage = '/';
  const signup = '/docs/accounts/accounts-billing/account-setup/create-your-new-relic-account/';

  if (pathname === homepage || pathname === signup) {
```

Comment (x8a) [reply src/components/SEO.js]: Thank you so much, @zstix! I wasn't sure about the location prop and whether it should be a relative path or an external link. I'll go ahead and merge your suggestion 👏🏻👏🏻👏🏻

**Review by jpvajda — APPROVED** [review]
I'm Ok that we merge this script  in, but I want to  run a lighthouse test after before / after deployment to understand impacts this script has on our site performance, for every script we add we should be doing that as a process @x8a  

I'll attach a lighthouse report to this PR and if you can plan on running this after deployment and reporting back the performance metrics that would be cool. 

Install the CLI here:
https://github.com/GoogleChrome/lighthouse

Run this command:
`lighthouse https://docs.newrelic.com --preset=desktop --view`

OR:
you can use this site https://web.dev/measure/

**Comment by jpvajda** [comment]
Light house stats for the site without this change:

Screen Shot 2021-09-02 at 12 34 28 PM

Screen Shot 2021-09-02 at 12 33 47 PM

**Comment by x8a** [comment]
@jpvajda, here's the report after deployment. Let me know if you want me to send you the full HTML report.
Screenshot 2021-09-06 at 10 32 40

**Comment by jpvajda** [comment]
Interesting, it went up... I'm thinking the lighthouse testing is sometimes a bit subjective depending on a lot of factors. But Given this is in a range of `+ / - 10 pts`  I think it's good to know

**Pull request merged**